src/templates/base/base.html

```diff
@@ -482,10 +482,19 @@
           element.hidden = false;
         });
 
-        // Language and Year switching
+        // Language, Year and ToC switching
         var languageYearSwitchers = document.querySelectorAll('.language-switcher select, .year-switcher select, .table-of-contents-switcher select');
         for (var i = 0; i < languageYearSwitchers.length; i++) {
           languageYearSwitchers[i].addEventListener('change', function(e) {
+
+            if (e.target.dataset.label) {
```

Comment:
Will this condition be true for all options? Should it only be true when the switcher has a label and the option has the `data-event` attribute?

Reply:
Had actually spotted this myself and fixed, but looks like I forgot to `git push`. Should be fixed now.